drivers/bus/isapnp/isapnp.c

```diff
@@ -822,7 +824,6 @@ IsaPnpCreateReadPortDORequirements(
     ULONG ResourceCount, ListSize, i;
     PIO_RESOURCE_REQUIREMENTS_LIST RequirementsList;
     PIO_RESOURCE_DESCRIPTOR Descriptor;
-    const ULONG Ports[] = { ISAPNP_WRITE_DATA, ISAPNP_ADDRESS };
     const ULONG ReadPorts[] = { 0x274, 0x3E4, 0x204, 0x2E4, 0x354, 0x2F4 };
```

Comment:
@disean Maybe for a future PR: Note that these ReadPorts values, and the IsaConfigPorts ones, are duplicated in the test: https://git.reactos.org/?p=reactos.git;a=blob;f=modules/rostests/unittests/isapnp/tests.c;hb=50a72881be93938639d25ca4d772594476667415#l16
I don't know whether it could be of interest to have these definitions shared in some way.